cli/prisma2/src/bin.ts

```diff
@@ -91,7 +99,20 @@ async function main(): Promise<number> {
     console.error(result)
     return 1
   }
-  console.log(result)
+
+  const checkResult = await checkpoint.check({
+    product: 'prisma',
+    version: packageJson.version,
+    cache_only: true,
+    disable: isCI,
+  })
+  if (checkResult && checkResult.outdated) {
+    console.error(
```

Comment:
Is this a real error?

Reply:
It's not a real error, but I figured it's best to print to stderr, since we might use stdout for something else like generating a result.